example/java/TransactionExample.java

```diff
@@ -80,7 +80,7 @@ public static void main(String[] args) {
             .createAsset("dollar", "ru", (short)2).build();
 
         // sign transaction and get its binary representation (Blob)
-        ByteVector txblob = protoTxHelper.signAndAddSignature(utx, keys).blob();
+        ByteVector txblob = protoTxHelper.signAndAddSignature(utx, keys).finish().blob().blob();
```

Comment:
why do you take blob twice?
If this is the result of some changes in our bindings, we should change our wrapper so that the interface does not have this ugly blob thing

Reply:
Maybe should we use **get** prefixes for getters?

Reply:
Or add method `bytes()` or `vector()` or something like this to `Crypto::Blob`?